docs/dev-guide/contributing.md

```diff
@@ -185,4 +214,17 @@ npx webpack-bundle-analyzer build/app-stats.json
 
 ### Kotlin
 
-For Kotlin code we use the [standard convention](https://kotlinlang.org/docs/coding-conventions.html) and limit line length to 120 characters. We use `ktlint` to enforce formatting.
+- For Kotlin code we use the [standard convention](https://kotlinlang.org/docs/coding-conventions.html) and limit line length to 120 characters. We use `ktlint` to enforce formatting.
+
+## ✅ Code Reviews
+
+- **Submit Your Contribution:** After completing your work, submit your contribution.
+- **Automatic Assignment:** Your pull request will be automatically assigned to appropriate reviewers based on the affected codebase and expertise required for the changes.
+- **Review Process:** Reviewers will carefully examine your code, checking for adherence to coding standards, correctness, performance and potential issues.
+- **Feedback and Iteration:** If any issues or suggestions are identified during the review, you'll receive feedback from the reviewers. Address any comments or concerns raised and make necessary revisions to your code.
+- **Test Your Changes:** Before finalizing your contribution, ensure that any relevant tests are run to verify the correctness and functionality of your changes.
```

Comment:
Relevant testing and linting should be done locally before even opening a PR. 

```suggestion
- **Automated tests:** Once the PR is in a good state, a team member will trigger the automated tests. The PR needs to merge cleanly on top of master, and test failures or issues discovered at this stage will need to be addressed before the PR is approved for merging.
```

Reply:
Thanks for your comments, all fixed @bgrozev